app/text_buffer.py

```diff
@@ -40,6 +40,9 @@ def __init__(self):
     self.rootGrammar = app.prefs.getGrammar(None)
     self.skipUpdateScroll = False
 
+    # Determines whether case matters when finding a selection
+    self.ignoreCaseFind = True
```

Comment:
how about adding 

'findIgnoreCase': True,

in ci_edit/app/default_prefs.py in the 'editor' section? It could be checked here with

app.prefs.prefs['editor'].get('findIgnoreCase')

That will allow us/users to set

"findIgnoreCase": false,

in ~/.ci_edit/prefs/editor.json to change it.